### PR TITLE
Genesis config rework

### PR DESCRIPTION
**Pull request by jingsting**
## Purpose
Some improvements to the genesis configuration pull request

## Approach
- changed totalStake to be calculated from the sum of arbits from the created block instead of `members.values.sum`
- updated the tests spec for genesis creation

## Testing
- tested the coinTotal for the generated genesis
- tested if the genesis from block Json method created the same blocks and chain params as the previous ones

## Tickets
* #1947 
* #1837

### CODE REVIEW

**Review by b-vennes — COMMENTED** [review]
It looks like there are a few things here and there that need to be cleaned up from debugging, but overall I think this looks good. Definitely be sure to update comments for methods where new parameters are added.

File: node/src/main/resources/application.conf [hunk 1]
```diff
@@ -35,6 +29,38 @@ bifrost {
...
+
+        # Difficulty the genesis block is created with
+        initialDifficulty = 1000000000000000000
+      }
+
+      fromBlockJson {
+        # file path for providing a genesis block from Json
+        providedJsonGenesisPath = "node/src/main/resources/toplnet-genesis.json"
+
+        # checksum to ensure the generated block is correct
```
[the remark this answers is not in the document]
Comment (jingsting) [reply node/src/main/resources/application.conf:59]: It's the same as the block id

File: node/src/test/scala/co/topl/api/transaction/ArbitTransferRPCSpec.scala [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package co.topl.api.transaction
 
 import co.topl.attestation.Address
+import co.topl.utils.DiskKeyRingTestHelper
```
[the remark this answers is not in the document]
Comment (jingsting) [reply node/src/test/scala/co/topl/api/transaction/ArbitTransferRPCSpec.scala:4]: Removed now

File: node/src/test/scala/co/topl/api/NameSpaceSpec.scala [hunk 3]
```diff
@@ -9,12 +9,13 @@ import co.topl.consensus.{ActorConsensusInterface, ActorForgerInterface, ActorKe
...
 import co.topl.nodeView.ActorNodeViewHolderInterface
 import co.topl.rpc.ToplRpcServer
+import co.topl.utils.TestSettings
 import co.topl.settings.{AppContext, AppSettings, StartupOpts}
 import io.circe.parser.parse
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
 
-class NameSpaceSpec extends AnyWordSpec with Matchers with RPCMockState {
+class NameSpaceSpec extends AnyWordSpec with Matchers with TestSettings with RPCMockState {
```
[the remark this answers is not in the document]
Comment (jingsting) [reply node/src/test/scala/co/topl/api/NameSpaceSpec.scala:18]: Removed it

**Pull request merged**